build.sh

```diff
@@ -45,6 +46,12 @@ case "$(uname -s)" in # adjust compilation option based on platform
         sys_cflags='-march=native -Wno-error=redundant-decls -Wno-error=unused-but-set-variable'
         sys_opts='--enable-kvm --disable-xen --disable-werror'
         ;;
+    FreeBSD)
```

Comment:
nit: I'd add this block as first case, so we have the reverse order of the default order-by-popularity (which then also happens to be alphabetical): Windows, MacOS [Darwin], Linux, BSD.

I'm not sure why the current order in this file is reverse to begin with (or why it's somewhat random).

Reply:
This is near the bottom now, and not near the top (like I had suggested). So I consider this unresolved and more problematic than before.

*Please do not resolve conversations which you didn't start; give reviewers the chance to resolve conversation when they believe the PR author has addressed all issues.*